onnxruntime/core/providers/cuda/cuda_profiler.cc

```diff
@@ -1,6 +1,6 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-#if !(defined(USE_ROCM) || defined(ENABLE_TRAINING))
+#if defined(ENABLE_CUDA_PROFILING) && !(defined(USE_ROCM) || defined(ENABLE_TRAINING))
```

Comment:
where is ENABLE_CUDA_PROFILING defined?

Reply:
it needs to be defined when onnxruntime_ENABLE_CUDA_PROFILING is defined right?

Reply:
It is defined in CMakeList.txt added in latest commit.